src/rospkg/common.py

```diff
@@ -45,13 +45,28 @@ class ResourceNotFound(Exception):
     A ROS filesystem resource was not found.
     """
 
-    def __init__(self, msg, ros_paths=None):
+    def __init__(self, msg, ros_paths=None, deps_sofar=None, deps_unavailable=None):
+        """
+        :type deps_sofar: [str]
+        :param deps_sofar: List of depended packages at the time the command
+                                        stopped due to this exception.
+        :type deps_unavailable: [str]
+        :param deps_unavailable: List of packages defined in the dependency but are not
+                                                  available on the platform.
```

Comment:
`deps_unavailable` is not used anywhere in this patch. Therefore I don't think it should be part of it.